teamcity: failed tests on master: test/TestLogic, test/TestLogic/default, test/TestLogic/default/crdb_internal, test/TestLogic/default/crdb_internal/base, test/TestLogic/distsql, test/TestLogic/distsql/crdb_internal, test/TestLogic/distsql/crdb_internal/base, test/TestLogic, test/TestLogic/default, test/TestLogic/default/crdb_internal, test/TestLogic/default/crdb_internal/base, test/TestLogic/distsql, test/TestLogic/distsql/crdb_internal, test/TestLogic/distsql/crdb_internal/base

The following tests appear to have failed:

[#449006](https://teamcity.cockroachdb.com/viewLog.html?buildId=449006):


```
--- FAIL: test/TestLogic (82.050s)
test_log_scope.go:78: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic867596294
test_log_scope.go:61: use -show-logs to present logs inline
--- FAIL: test/TestLogic/default (0.220s)
None
--- FAIL: test/TestLogic/default/crdb_internal (0.440s)
logic_test.go:2054: 
	testdata/logic_test/crdb_internal:208: select crdb_internal.node_executable_version()
	expected:
	    1.1-6
	    
	but found (query options: "") :
	    1.1-7  
	
--- FAIL: test/TestLogic/default/crdb_internal/base (0.130s)
logic_test.go:1032: testdata/logic_test/crdb_internal:213: too many errors encountered, skipping the rest of the input
------- Stdout: -------
E171218 23:27:34.871408 32563 sql/logictest/logic_test.go:2053  
testdata/logic_test/crdb_internal:208: select crdb_internal.node_executable_version()
expected:
    1.1-6
    
but found (query options: "") :
    1.1-7  
--- done: testdata/logic_test/crdb_internal with config default: 41 tests, 1 failures
--- done: testdata/logic_test/pgoidtype with config default: 60 tests, 0 failures
--- done: testdata/logic_test/partitioning_index with config default: 13 tests, 0 failures
--- FAIL: test/TestLogic/distsql (0.390s)
None
--- FAIL: test/TestLogic/distsql/crdb_internal (0.930s)
logic_test.go:2054: 
	testdata/logic_test/crdb_internal:208: select crdb_internal.node_executable_version()
	expected:
	    1.1-6
	    
	but found (query options: "") :
	    1.1-7  
	
--- FAIL: test/TestLogic/distsql/crdb_internal/base (0.280s)
logic_test.go:1032: testdata/logic_test/crdb_internal:213: too many errors encountered, skipping the rest of the input
--- FAIL: test/TestLogic (82.050s)
test_log_scope.go:78: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestLogic867596294
test_log_scope.go:61: use -show-logs to present logs inline
--- FAIL: test/TestLogic/default (0.220s)
None
--- FAIL: test/TestLogic/default/crdb_internal (0.440s)
logic_test.go:2054: 
	testdata/logic_test/crdb_internal:208: select crdb_internal.node_executable_version()
	expected:
	    1.1-6
	    
	but found (query options: "") :
	    1.1-7  
	
--- FAIL: test/TestLogic/default/crdb_internal/base (0.130s)
logic_test.go:1032: testdata/logic_test/crdb_internal:213: too many errors encountered, skipping the rest of the input
------- Stdout: -------
E171218 23:27:34.871408 32563 sql/logictest/logic_test.go:2053  
testdata/logic_test/crdb_internal:208: select crdb_internal.node_executable_version()
expected:
    1.1-6
    
but found (query options: "") :
    1.1-7  
--- done: testdata/logic_test/crdb_internal with config default: 41 tests, 1 failures
--- done: testdata/logic_test/pgoidtype with config default: 60 tests, 0 failures
--- done: testdata/logic_test/partitioning_index with config default: 13 tests, 0 failures
--- FAIL: test/TestLogic/distsql (0.390s)
None
--- FAIL: test/TestLogic/distsql/crdb_internal (0.930s)
logic_test.go:2054: 
	testdata/logic_test/crdb_internal:208: select crdb_internal.node_executable_version()
	expected:
	    1.1-6
	    
	but found (query options: "") :
	    1.1-7  
	
--- FAIL: test/TestLogic/distsql/crdb_internal/base (0.280s)
logic_test.go:1032: testdata/logic_test/crdb_internal:213: too many errors encountered, skipping the rest of the input

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/logictest/testdata/logic_test/crdb_internal
+++ b/pkg/sql/logictest/testdata/logic_test/crdb_internal
@@ -208,7 +208,7 @@ select crdb_internal.set_vmodule('')
 query T
 select crdb_internal.node_executable_version()
 ----
-1.1-6
+1.1-7
 
 query ITTT colnames
 select node_id, component, field, regexp_replace(regexp_replace(value, '^\d+$', '<port>'), e':\\d+', ':<port>') as value from crdb_internal.node_runtime_info
@@ -252,4 +252,4 @@ select * from crdb_internal.ranges
 query T
 select crdb_internal.node_executable_version()
 ----
-1.1-6
+1.1-7

